truffle.js

```diff
@@ -52,11 +51,18 @@ module.exports = {
     },
     rinkeby: {
       provider: function() {
-        return new HDWalletProviderPriv(privkeys, 'https://rinkeby.infura.io/' + token)
+        return new HDWalletProvider({ privateKeys, providerOrUrl: 'https://rinkeby.infura.io/' + token })
       },
       network_id: 4,
       from: '0x6239ff1040e412491557a7a02b2cbcc5ae85dc8f',
     },
+    xdai: {
+      provider: function() {
+        return new HDWalletProvider({ privateKeys, providerOrUrl: 'https://dai.poa.network' })
+      },
+      network_id: 100,
+      from: '0x6239ff1040e412491557a7a02b2cbcc5ae85dc8f',
```

Comment:
Obviously I am not able to deploy from your address.

I deployed `Kudos` `from` my own address, and then transferred it to the address displayed here.

I am committing this with your address shown here for consistency's sake, considering the rest of this config file.

Reply:
thx